reinforcement_learning/rl_cartpole_batch_coach/rl_cartpole_batch_coach.ipynb

```diff
@@ -0,0 +1,712 @@
+{
```

Comment:
<p>"episodes of trajectories" -- what does this mean?</p><p> </p><p>"We start..." How about "We start from a random policy that generates the same action probabilities regardless of the state features."?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/926/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>resolved</p>